### PR TITLE
V1 Migration

### PR DESCRIPTION
**Pull request by netboy3**
Hi. Just completed a full migration of your VCV plugin to Rack V1. It builds with no warnings on Linux. I've also tested functionality on Linux and all seems to be working well. Please note this is a full migration so the plugin runs now in V1 native mode and not compatibility mode (rack0). Thanks again for your work on this plugin.

### CODE REVIEW

**Comment by elijahfhopp** [comment]
Excited about this! I love 21kHz. Probably one of the nicest looking, easy to use, and fun VCV Rack modules out there. Thanks for all the work you put into it, @netboy3!

File: src/21kHz.cpp [hunk 1]
```diff
@@ -1,19 +1,12 @@
 #include "21kHz.hpp"
 
-
-Plugin *plugin;
-
+Plugin *pluginInstance;
 
 void init(Plugin *p) {
```
Comment (qno) [inline src/21kHz.cpp:5]: Indentation looks a bit weird in this method!

Comment (netboy3) [reply src/21kHz.cpp:5]: I agree. This is not the only file with indentation issues. Some lines were hard tabbed and some soft tabbed. This is not something I did to the code but rather was there before. If you view the code in a proper editor (I use Atom), physical screen spacing is compensated and all looks fine. If I end up adopting this module, I'll clean up all the soft/hard indentation mess.

Comment (elijahfhopp) [reply src/21kHz.cpp:5]: @netboy3, are you still considering adopting this module? I would love to see some builds as I haven't made the time to setup a VCV Rack development environment. 

I think I am going to send 21kHz an email. I got his email off of the VCV Rack website. It's jakehuryn@gmail.com if you care to send him one too. I hope it's not dead like his Github account. :sweat_smile: 

EDIT: FOUND YOUR BUILDS! Thanks, m8. I love this module and can't wait to use it with V1.

Comment (netboy3) [reply src/21kHz.cpp:5]: UR Welcome. I probably am going to adopt it in the next few weeks. I usually don't want to step on the developer's toes - it's really their creation, but I believe I've done due diligence in trying to track the dev down...

**Pull request merged**